runtime/oti/j9java8packages.gperf

```diff
@@ -0,0 +1,2068 @@
+sun/util/resources/tr
+sun/util/resources/et
+sun/util/resources/zh
+sun/util/resources/hi
+sun/util/resources/nl
+sun/util/resources/hr
+sun/util/resources/ar
+sun/util/resources/fi
+sun/util/resources/no
+sun/util/resources/sk
+sun/util/resources/ml
+sun/util/resources/vi
+sun/util/resources/en
+sun/util/resources/th
+sun/util/resources/lt
+sun/util/resources/kn
+sun/util/resources/te
+sun/util/resources/pl
+sun/util/resources/or
+sun/util/resources/sh
+sun/util/resources/pt
+sun/util/resources/is
+sun/util/resources/sr
+sun/util/resources/mr
+sun/util/resources/bn
+sun/util/resources/sl
+sun/util/resources/fr
+sun/util/resources/kk
+sun/util/resources/gu
+sun/util/resources/nb
+sun/util/resources/cs
+sun/util/resources/be
+sun/util/resources/ca
+sun/text/resources/el
+sun/text/resources/uk
+sun/text/resources/da
+sun/text/resources/sv
+sun/text/resources/hu
+sun/text/resources/bg
+sun/text/resources/ms
+sun/text/resources/it
+sun/text/resources/de
+sun/text/resources/ru
+sun/text/resources/lv
+sun/text/resources/iw
+sun/text/resources/ja
+sun/text/resources/ga
+sun/text/resources/es
+sun/text/resources/sq
+sun/text/resources/ro
+sun/text/resources/ko
+sun/text/resources/in
+sun/text/resources/mk
+sun/text/resources/mt
+sun/text/resources/tr
+sun/text/resources/et
+sun/text/resources/zh
+sun/text/resources/hi
+sun/text/resources/nl
+sun/text/resources/hr
+sun/text/resources/ar
+sun/text/resources/fi
+sun/text/resources/no
+sun/text/resources/sk
+sun/text/resources/vi
+sun/text/resources/en
+sun/text/resources/th
+sun/text/resources/lt
+sun/text/resources/pl
+sun/text/resources/pt
+sun/text/resources/is
+sun/text/resources/sr
+sun/text/resources/sl
+sun/text/resources/fr
+sun/text/resources/cs
+sun/text/resources/be
+sun/text/resources/ca
+jdk/internal/dynalink
+jdk/internal/dynalink/linker
+jdk/internal/dynalink/support
+jdk/internal/dynalink/beans
+jdk/nashorn/tools
+jdk/nashorn/api/scripting
+jdk/nashorn/internal/scripts
+jdk/nashorn/internal/runtime
+jdk/nashorn/internal/runtime/arrays
+jdk/nashorn/internal/runtime/linker
+jdk/nashorn/internal/runtime/logging
+jdk/nashorn/internal/runtime/regexp
+jdk/nashorn/internal/runtime/regexp/joni
+jdk/nashorn/internal/runtime/regexp/joni/encoding
+jdk/nashorn/internal/runtime/regexp/joni/ast
+jdk/nashorn/internal/runtime/regexp/joni/exception
+jdk/nashorn/internal/runtime/regexp/joni/constants
+jdk/nashorn/internal/runtime/options
+jdk/nashorn/internal/runtime/events
+jdk/nashorn/internal/objects
+jdk/nashorn/internal/objects/annotations
+jdk/nashorn/internal/codegen
+jdk/nashorn/internal/codegen/types
+jdk/nashorn/internal/lookup
+jdk/nashorn/internal
+jdk/nashorn/internal/ir
+jdk/nashorn/internal/ir/annotations
+jdk/nashorn/internal/ir/debug
+jdk/nashorn/internal/ir/visitor
+jdk/nashorn/internal/parser
+com/ibm/jvm/format
+com/ibm/jvm/trace
+com/ibm/jvm/trace/format/api
+javax/xml/crypto/enc
+javax/xml/crypto/enc/dom
+javax/xml/crypto/enc/keyinfo
+javax/xml/crypto/enc/spec
+com/sun/nio/zipfs
+java
```

Comment:
This doesn't seem right, are you including directories that don't contain classes?